pkg/cmd/run/watch/watch.go

```diff
@@ -0,0 +1,228 @@
+package watch
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+	"runtime"
+	"time"
+
+	"github.com/cli/cli/api"
+	"github.com/cli/cli/internal/ghrepo"
+	"github.com/cli/cli/pkg/cmd/run/shared"
+	"github.com/cli/cli/pkg/cmdutil"
+	"github.com/cli/cli/pkg/iostreams"
+	"github.com/cli/cli/utils"
+	"github.com/spf13/cobra"
+)
+
+const defaultInterval int = 3
```

Comment:
It might be nice to add a comment or change the name to indicate that the interval is in seconds.

Reply:
I think it's okay to be without a unit; I just always assume seconds anyway (and if it's different, that should be pointed out).

Reply:
i agree with mislav that seconds is usually implied